rebar.config

```diff
@@ -1,10 +1,10 @@
 {lib_dirs,["deps"]}.
 {sub_dirs, ["rel"]}.
 
-{deps, [
-    {lager, "2.0", {git, "git://github.com/basho/lager", {tag, "2.0.3"}}}
-]}.
+%%{deps, [
+%%    {lager, "2.0", {git, "git://github.com/basho/lager", {tag, "2.0.3"}}}
+%%]}.
 
-{erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform}, {i,"../include"}]}.
+%%{erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform}, {i,"../include"}]}.
```

Comment:
Hi, I pushed that change by mistake, but:

1. the current code does not use lager.
2. antidote_utils does not compile (erlang19) if that's not commented out.
3. that's a different version than the one used by antidote.

I think we should remove those lines, do you?
Otherwise I can uncomment them back.